### ISSUE TITLE
Not authorizing with Google Container Engine

### ISSUE THREAD

**Issue by mark9white** [boz/kail#1]
Thanks for this utility.

Note that it doesn't work with Google Container Engine:
```
kail: error: Error configuring kubernetes connection: No Auth Provider found for name "gcp"
```
Looks like you may need to register the gcp plugin (k8s.io/client-go/plugin/pkg/client/auth/gcp) - see: https://github.com/kubernetes/kubernetes/pull/41532

**Comment by boz** [comment]
Thanks for taking a look and for the report.  I'll get to it shortly.

**Comment by boz** [comment]
fixed by #2
